features/i8n.feature

```diff
@@ -0,0 +1,118 @@
+Feature: Pro supports multiple languages
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    @uses.config.contract_token
+    Scenario Outline: Pro client's commands run successfully
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        ## Change the locale
+        When I run `locale-gen uk_UA.UTF-8` with sudo
+        And I run `update-locale LANG=uk_UA.UTF-8` with sudo
+        And I reboot the machine
+        And I run `cat /etc/default/locale` as non-root
+        Then stdout matches regexp:
+        """
+        LANG=uk_UA.UTF-8
+        """
```

Comment:
We should also run a couple commands to make sure their output is non-english. `lscpu` and `apt update` are good candidates